ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -703,13 +703,14 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
-- as: aws-ipi-private-fips-f28-disasterrecovery
-  cron: 11 11 20 * *
+- as: aws-ipi-private-fips-f4-disasterrecovery
+  cron: 11 11 4,8,12,16,20,24,28 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       FIPS_ENABLED: "true"
+      TEST_IMPORTANCE: LEVEL0
     test:
     - ref: idp-htpasswd
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
```

Comment:
Do we really want to run the other tests in this chain with frequency f4, or we only want to run LEVEL0 test cases from ginkgo ?

Reply:
We just want to get a results of LEVEL0 test cases for DR, this change is only temporary to get a test pass rate, and if we get the expected results, we will revert to the previous configuration.